contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -216,6 +218,8 @@ func NewSchedulerServer() *SchedulerServer {
 		kubeletEnableDebuggingHandlers: true,
 		containPodResources:            true,
 		nodeRelistPeriod:               defaultNodeRelistPeriod,
+		conntrackTCPTimeoutEstablished: 86400, // 1 day (1/5 default), copied from cmd/kube-proxy/app/server.go
```

Comment:
the other way to do it would be to extract the constant, used in kube-proxy, into a public const and then reference it here. no pointer magic.